### PR TITLE
dash/sync-docs-improvements

### PR DESCRIPTION
**Pull request by Vazonik**
Improved synchronization docs.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206366652699840

### CODE REVIEW

File: docs/dashboards/html-component.md [hunk 1]
```diff
@@ -115,5 +115,11 @@ The most common way to style the HTML component is to use the CSS and utilize th
 If some of the HTML elements are used frequently or requires to be more customable, it is possible to create a custom component that will simplify the configuration. The custom component can be created by extending the basic HTML component.
```
Comment (jomar-honsi) [inline docs/dashboards/html-component.md:115]: requires -> require

File: docs/dashboards/html-component.md [hunk 2]
```diff
@@ -115,5 +115,11 @@ The most common way to style the HTML component is to use the CSS and utilize th
 If some of the HTML elements are used frequently or requires to be more customable, it is possible to create a custom component that will simplify the configuration. The custom component can be created by extending the basic HTML component.  
 More in the [Custom Component](https://www.highcharts.com/docs/dashboards/custom-component) section.
 
+
+### Components synchronization
+
+HTML Component is the only component that does not have predefined synchronization types for itself (mainly because it cannot be connected to data pool). However, you can define your own sync types as needed. You can read more about it [here](https://www.highcharts.com/docs/dashboards/synchronize-components#custom-synchronization).
```
Comment (jomar-honsi) [inline docs/dashboards/html-component.md:121]: Suggested rewrite: The HTML Component is the only component that does not support synchronization types.

File: docs/dashboards/kpi-component.md [hunk 3]
```diff
@@ -163,8 +163,7 @@ The `value` can be customized by:
     ```
 
 ## Sync with other components
-The KPI Component allows user to sync the component with other components in dashboard or group.
-You can find more information about it in the [sync chapter](https://www.highcharts.com/docs/dashboards/synchronize-components).
+The KPI Component allows user to sync the component with other components in dashboards. You can find more information about it in the [sync chapter](https://www.highcharts.com/docs/dashboards/synchronize-components).
```
Comment (jomar-honsi) [inline docs/dashboards/kpi-component.md:166]: allows user -> _allows the user_ OR _allows users_

File: docs/dashboards/navigator-component.md [hunk 4]
```diff
@@ -43,6 +43,12 @@ Dashboard.board('container', {
 
 
 
+## Components synchronization
+
+The Navigator Component can be synced with other components in dashboards. Two synchronization types are predefined for the Navigator Component: [`crossfilter`](https://api.highcharts.com/dashboards/#interfaces/Dashboards_Components_NavigatorComponent_NavigatorComponentOptions.SyncOptions#crossfilter) and [`extremes`](https://api.highcharts.com/dashboards/#interfaces/Dashboards_Components_NavigatorComponent_NavigatorComponentOptions.SyncOptions#extremes). You can find more information about it in the [sync chapter](https://www.highcharts.com/docs/dashboards/synchronize-components).
```
Comment (jomar-honsi) [inline docs/dashboards/navigator-component.md:48]: ... components in _dashboards_ ... Dashboards is a trade mark, so should arguably be written Dashboards.

File: docs/dashboards/synchronize-components.md [hunk 5]
```diff
@@ -6,8 +6,10 @@ and highlighting specific data.
 
 <iframe style="width: 100%; height: 470px; border: none;" src=https://www.highcharts.com/samples/embed/dashboards/demo/minimal allow="fullscreen"></iframe>
 
+
 ## How to synchronize the components?
-To synchronize components you have to specify which event you want to synchronize between each component, as well as they have to use the same connector defined in the [dataPool](https://www.highcharts.com/docs/dashboards/data-handling).
+
+To synchronize components you have to specify which event you want to synchronize between each component, as well as they have to use the same connector defined in the [dataPool](https://www.highcharts.com/docs/dashboards/data-handling) which is needed for all the predefined synchronization types to work.
```
Comment (jomar-honsi) [inline docs/dashboards/synchronize-components.md:12]: Long and cumbersome sentence, consider splitting it up.

**Review by jomar-honsi — CHANGES_REQUESTED** [review]

File: docs/dashboards/html-component.md (hunk 2 above)
Comment (karolkolodziej) [inline docs/dashboards/html-component.md:121]: Technically `HTMLComponent extends Component ` which means that the connector can exist. 
So I'm not sure if the statement in the parentheses is true-[ see demo.](https://jsfiddle.net/BlackLabel/k9w5yoms/)

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

**Review by karolkolodziej — APPROVED** [review]

**Review by sebastianbochan — APPROVED** [review]

**Review by jomar-honsi — APPROVED** [review]

File: docs/dashboards/html-component.md [hunk 6]
```diff
@@ -112,8 +112,14 @@ The most common way to style the HTML component is to use the CSS and utilize th
 
 ### Extending the HTML component
 
-If some of the HTML elements are used frequently or requires to be more customable, it is possible to create a custom component that will simplify the configuration. The custom component can be created by extending the basic HTML component.  
+If some of the HTML elements are used frequently or require to be more customable, it is possible to create a custom component that will simplify the configuration. The custom component can be created by extending the basic HTML component.
```
Comment (stitot) [inline docs/dashboards/html-component.md:115]: customable = customizable

File: docs/dashboards/kpi-component.md [hunk 7]
```diff
@@ -163,8 +163,7 @@ The `value` can be customized by:
     ```
 
 ## Sync with other components
-The KPI Component allows user to sync the component with other components in dashboard or group.
-You can find more information about it in the [sync chapter](https://www.highcharts.com/docs/dashboards/synchronize-components).
+The KPI Component allows users to sync the component with other components in dashboards. You can find more information about it in the [sync chapter](https://www.highcharts.com/docs/dashboards/synchronize-components).
```
Comment (stitot) [inline docs/dashboards/kpi-component.md:166]: chapter = article

File: docs/dashboards/navigator-component.md [hunk 8]
```diff
@@ -43,6 +43,12 @@ Dashboard.board('container', {
 
 
 
+## Components synchronization
+
+The Navigator Component can be synced with other components in Dashboards. Two synchronization types are predefined for the Navigator Component: [`crossfilter`](https://api.highcharts.com/dashboards/#interfaces/Dashboards_Components_NavigatorComponent_NavigatorComponentOptions.SyncOptions#crossfilter) and [`extremes`](https://api.highcharts.com/dashboards/#interfaces/Dashboards_Components_NavigatorComponent_NavigatorComponentOptions.SyncOptions#extremes). You can find more information about it in the [sync chapter](https://www.highcharts.com/docs/dashboards/synchronize-components).
```
Comment (stitot) [inline docs/dashboards/navigator-component.md:48]: chapter = article

File: docs/dashboards/datagrid-component.md [hunk 9]
```diff
@@ -157,7 +157,7 @@ Note that you also need to import modules to use the appropriate modifiers. For
 
 ## Components synchronization
 
-One of the many available options for the DataGrid Component is the [`sync` option](https://api.highcharts.com/dashboards/#interfaces/Dashboards_Components_DataGridComponent_DataGridComponentOptions.Options#sync), which allows setting the synchronization of component states with each other.
+One of the many available options for the DataGrid Component is the [`sync` option](https://api.highcharts.com/dashboards/#interfaces/Dashboards_Components_DataGridComponent_DataGridComponentOptions.Options#sync), which allows setting the synchronization of component states with each other. You can find more information about it in the [sync chapter](https://www.highcharts.com/docs/dashboards/synchronize-components).
```
Comment (stitot) [inline docs/dashboards/datagrid-component.md:160]: chapter = article

File: docs/dashboards/highcharts-component.md [hunk 10]
```diff
@@ -177,6 +177,21 @@ columnAssignment: [{
 ```
 <iframe style="width: 100%; height: 600px; border: none;" src=https://www.highcharts.com/samples/embed/dashboards/components/highcharts-column-assignment-keys-data allow="fullscreen"></iframe>
 
+
+## Components synchronization
+
+One of the many available options for the Highcharts Component is the [`sync` option](https://api.highcharts.com/dashboards/#interfaces/Dashboards_Components_HighchartsComponent_HighchartsComponentOptions.Options#sync), which allows setting the synchronization of component states with each other. You can find more information about it in the [sync chapter](https://www.highcharts.com/docs/dashboards/synchronize-components).
```
Comment (stitot) [inline docs/dashboards/highcharts-component.md:183]: chapter = article

**Review by stitot — APPROVED** [review]

File: docs/dashboards/kpi-component.md [hunk 11]
```diff
@@ -163,7 +163,7 @@ The `value` can be customized by:
     ```
 
 ## Sync with other components
-The KPI Component allows users to sync the component with other components in dashboards. You can find more information about it in the [sync chapter](https://www.highcharts.com/docs/dashboards/synchronize-components).
+The KPI Component allows users to sync the component with other components in dashboards. You can find more information about it in the [sync article](https://www.highcharts.com/docs/dashboards/synchronize-components).
```
Comment (jomar-honsi) [inline docs/dashboards/kpi-component.md:166]: Knitpicking: Conistently write Dashboards with the first letter capitalised.

**Review by jomar-honsi — APPROVED** [review]

**Pull request merged**